### DISCUSSION TITLE
dtcw improvement

### DISCUSSION THREAD

**Discussion by joel-loewenstein** [docToolchain/docToolchain#541] [Ideas]
Hi, 
based on your [FOSDEM 21 lecture](https://fosdem.org/2021/schedule/event/ttddoctoolchain/) i was keen on testing it on mac (where i predominantly work on) and found some topics and have questions. Would you already want issues for that? Or prefer disucssion here?

**Comment by rdmueller** [comment]
I've created a label for 'next-gen' issues, so I guess the issue tracker would be the right choice for specific issues.
Other things which are not clearly bugs can be discussed here

**Reply by rdmueller** [reply]
don't hesitate to send your feedback - it doesn't matter if it is written down in the wrong tool :-)

**Comment by joel-loewenstein** [comment]
I created #544 to improve dtcw with an interactive mode

**Comment by uniqueck** [comment]
Where can i find the repository for the dtcw? I would like to provide a pull request for my issue?

**Comment by rdmueller** [comment]
to make it easy downloadable, dtcw is located in https://github.com/docToolchain/docToolchain.github.io
this results in doctoolchain.github.io/dtcw as download-url

**Comment by rdmueller** [comment]
PS: there is a development section in the new manual (see 2.0.x in the url) where I want to collect those infos:
https://doctoolchain.github.io/docToolchain/v2.0.x/#_development

If something is missing (and it surely is), we should add it here
